api/v1/controllers/subjects.js

```diff
@@ -198,26 +199,20 @@ module.exports = {
   /**
    * GET /subjects
    *
-   * Finds zero or more subjects and sends them back in the response.
+   * Finds zero or more subjects and sends them back in the response. Uses
+   * cached response if env var SUBJECT_CACHE_EXPIRY_IN_SECS > 0 and cached
+   * response is available.
    *
    * @param {IncomingMessage} req - The request object
    * @param {ServerResponse} res - The response object
    * @param {Function} next - The next middleware function in the stack
    */
   findSubjects(req, res, next) {
     validateTags(null, req.swagger.params);
-    if (featureToggles.isFeatureEnabled('getSubjectFromCache')) {
```

Comment:
If we're removing the existing subject caching, I think there's also code in cache/models that can be removed.

Reply:
yes, will remove that fn in next pr